components/yaml/introduction.rst

```diff
@@ -148,6 +148,21 @@ error if something goes wrong by adding the filename to the message.
 
 .. _components-yaml-dump:
 
+Objects for Mappings
+....................
+
+.. versionadded:: 2.7
```

Comment:
Ah indeed, this was a bit inconsistent. But as we should prefer the static API 2.7 is a good choice here.